docs/roadmap/intro.md

```diff
@@ -22,27 +22,28 @@ Over the course of the two-year grant, we are aiming to transform the Jupyter ec
 
 ## Action items
 
-### :runner: In the works
```

Comment:
Thanks I saw it the other day and noted it mentally to fix it 🎉  but you read my mind